src/Engine/GameEvents/MapChangeEvent.java

```diff
@@ -61,13 +61,17 @@ public void performEvent(MapMaster gameMap)
   @Override
   public XYCoord getStartPoint()
   {
-    return changes.get(0).where;
+    if( changes.size() > 0 )
+      return changes.get(0).where;
+    return null;
```

Comment:
What was making this cause a crash?

Reply:
I wouldn't count on being able to see the lack of weather changing. There can be a lot going on, and the user won't necessarily know what "correct" is.
That said, this probably isn't going to be a critical thing either way. I'd prefer to at least have a warning though.